modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractKotlinCodegen.java

```diff
@@ -261,6 +270,23 @@ public void setEnumPropertyNaming(final String enumPropertyNamingType) {
         }
     }
 
+    /**
+     * Sets the serialization engine for Kotlin
+     *
+     * @param enumSerializationLibrary The string representation of the serialization library as defined by {@link CodegenConstants.SERIALIZATION_LIBRARY_TYPE}
```

Comment:
@prisoneroftech should it be `{@link org.openapitools.codegen.CodegenConstants.SERIALIZATION_LIBRARY_TYPE}` instead (so as to fix the Java warning)?

Reply:
The javadoc tool is not the same depending on the JVM (this can explain why it fails in CI and why it is not failing on your machine)